frontend/src/app/application-forms/fields/file-upload.component.spec.ts

```diff
@@ -38,14 +38,23 @@ describe('FileUploadComponent', () => {
     component.onCompleteItem(null, null, null, null);
     expect(component.fileUploadService.numberOfFiles).toEqual(1);
   });
+  // testing below
+  it('should activate clickInput', async(() => {
+    spyOn(component, 'clickInput');
+    const button = fixture.debugElement.nativeElement.querySelector('.upload-file-btn');
+    button.click();
+    fixture.whenStable().then(() => {
+      expect(component.clickInput).toHaveBeenCalled();
+    });
+  }));
 
-  it('should update field after file is added', () => {
+  xit('should update field after file is added', () => {
```

Comment:
Instead of skipping this test can we rework/replace/remove it?

Reply:
@briandavidson - I can simply remove it. Mike had suggested skipping it for now and doing manual testing in dev. I do like the idea of removing it for now. We could always add it back in if we decide to go back and write one.

Reply:
@Dmac26 looks like either this still needs to be addressed, or if you're going with mikes suggestion you can simply resolve this conversation.